plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -371,6 +371,10 @@ const { state: cartItemState } = store(
 
 				return formatPriceWithCurrency( totalLinePrice, itemCurrency );
 			},
+
+			get isAvailableInBackorder(): boolean {
+				return cartItemState.cartItem.show_backorder_badge;
+			},
```

Comment:
Theoretically, I believe we could use directly `context...show_backorder_badge` in the directive. But, if I am not mistaken, we need to use derived state until this is fixed: https://github.com/woocommerce/woocommerce/blob/923651affc42007b94a29e62f680507a5680267f/plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts#L198-L210